assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -476,8 +476,8 @@ che.singleport.wildcard_domain.ipless=false
 
 # Docker image of Che plugin broker app that resolves workspace tooling configuration and copies
 # plugins dependencies to a workspace
-che.workspace.plugin_broker.init.image=eclipse/che-init-plugin-broker:v0.24
```

Comment:
Is it backward compatible? What if someone uses it? should we add aliases for this kind of change?

Reply:
This is in a section called "Experimental properties". So I would not sweat over breaking back compat here.

Reply:
A migration step is necessary if a user is specifying their own plugin brokers, which is a good point and should be documented. What will currently happen is
- Their custom brokers are ignored.
- The upstream built brokers would be used.

For users deploying with the default properties, no there aren't issues with moving versions.